internal/controlplane/handlers_repositories.go

```diff
@@ -364,16 +364,16 @@ func (s *Server) ListRemoteRepositoriesFromProvider(
 		return nil, err
 	}
 
-	zerolog.Ctx(ctx).Debug().
-		Str("provider", in.Provider).
-		Str("projectID", projectID.String()).
-		Msgf("listing repositories for provider: %s", in.Provider)
-
 	provider, err := getProviderFromRequestOrDefault(ctx, s.store, in, projectID)
 	if err != nil {
 		return nil, providerError(fmt.Errorf("provider error: %w", err))
 	}
 
+	zerolog.Ctx(ctx).Debug().
+		Str("provider", provider.Name).
+		Str("projectID", projectID.String()).
+		Msgf("listing repositories for provider: %s", provider.Name)
```

Comment:
Perhaps we can skip the provider name in `Msgf` since we already log it as a parameter to the same log message.